src/main/scala/tech/ant8e/sbt/i18n/BundleEmitter.scala

```diff
@@ -42,31 +53,47 @@ case class BundleEmitter(config: Config, packageName: String, breakOnMissingKeys
 
   private def toScalaType(paramType: ParamType) =
     paramType match {
-      case StringParam => "String"
-      case DateParam   => "java.util.Date"
-      case DoubleParam => "Double"
-      case LongParam   => "Long "
+      case AnyParam     => "Any"
```

Comment:
Changed `StringType` to `AnyType` as `{0}` pattern will be transformed to the regular toString invocation.